File: sec1/python/primes.py
def generate_primes(no=100):
    primes = []
    for i in range(1, no+1):
        if i != 1:
            is_prime = True
            for prime in primes:
                if not i%prime:
                    is_prime = False
                    break
            if is_prime:
                primes.append(i)
    return primes

print(generate_primes())

File: sec1/js/primes.js
function generatePrimes(no=100) {
    var primes = [];
    for (var i = 1; i <= no; i++) {
        if (i !== 1) {
            isPrime = true;
            for (var j = 0; j < primes.length; j++) {
                if (i % primes[j] == 0) {
                    isPrime = false;
                    break;
                }
            }
            if (isPrime) {
                primes.push(i);
            }
        }
    }

    return primes;
}

console.log(generatePrimes());